tests/lib/check/check_loader_test.py

```diff
@@ -232,51 +217,56 @@ def test_load_checks_to_execute_with_service_list(
         compliance_frameworks = None
         categories = None
 
-        with patch(
-            "prowler.lib.check.checks_loader.CheckMetadata.list_by_service",
-            return_value={S3_BUCKET_LEVEL_PUBLIC_ACCESS_BLOCK_NAME},
-        ):
-            assert {S3_BUCKET_LEVEL_PUBLIC_ACCESS_BLOCK_NAME} == load_checks_to_execute(
-                bulk_checks_metatada,
-                bulk_compliance_frameworks,
-                checks_file,
-                check_list,
-                service_list,
-                severities,
-                compliance_frameworks,
-                categories,
-                self.provider,
-            )
+        assert {S3_BUCKET_LEVEL_PUBLIC_ACCESS_BLOCK_NAME} == load_checks_to_execute(
+            bulk_checks_metatada,
+            bulk_compliance_frameworks,
+            checks_file,
+            check_list,
+            service_list,
+            severities,
+            compliance_frameworks,
+            categories,
+            self.provider,
+        )
 
     def test_load_checks_to_execute_with_compliance_frameworks(
         self,
     ):
-        bulk_checks_metatada = {
-            S3_BUCKET_LEVEL_PUBLIC_ACCESS_BLOCK_NAME: self.get_custom_check_metadata()
+        bulk_compliance_frameworks = {
+            "soc2_aws": Compliance(
+                Framework="SOC2",
+                Provider="aws",
+                Version="2.0",
+                Description="This CIS Benchmark is the product of a community consensus process and consists of secure configuration guidelines developed for Azuee Platform",
+                Requirements=[
+                    Compliance_Requirement(
+                        Checks=[S3_BUCKET_LEVEL_PUBLIC_ACCESS_BLOCK_NAME],
+                        Id="",
+                        Description="",
+                        Attributes=[],
+                    )
+                ],
+            ),
         }
-        bulk_compliance_frameworks = None
+        bulk_checks_metatada = None
         checks_file = None
         check_list = []
         service_list = []
         severities = []
```

Comment:
If they are optional you can remove them right?

Reply:
Review the rest of tests.

Reply:
Done!!